components/00-mixins/opening-hours/_opening-hours.scss

```diff
@@ -28,14 +28,14 @@
   > .openinghours {
     @include theme('border-color', 'color-primary--lighten-3', 'opening-hours-border');
 
-    padding-right: $month-day-size / 2;
+    padding-right: $month-day-size *.5;
     padding-bottom: 0;
     border-right: 2px solid;
   }
 
   .openinghours--details {
 
-    top: -$month-day-size / 2;
+    top: -$month-day-size *.5;
```

Comment:
Add space between * and .5. Makes it clearer that it is a calculation instead of a wildcard.